app/src/main/res/values/strings.xml

```diff
@@ -283,11 +283,16 @@
     <string name="city">City</string>
     <string name="checkbox_message">Ticket holder is the same person as ticket buyer</string>
     <string name="error_fetching_attendees">Error fetching attendees</string>
+    <string name="used_eventyay_before">Used Eventyay before?</string>
+    <string name="sign_in_order_text">for fast and easy ordering</string>
+    <string name="sign_in_to_order_ticket">Please sign in to order the ticket</string>
+    <string name="unverified_user_order_message">Unverified account cannot get free tickets. Please verify your account</string>
```

Comment:
It should be a server-side implementation. Please check comments on PR #1931

Reply:
Removed

Reply:
Buid is failing